experiments/verify_fasm_to_bels_yosys.yaml

```diff
@@ -1,5 +1,8 @@
 designs:
+    - base/
```

Comment:
I don't know if everyone in the base folder compares as equivalent.  Does it?  Let's only include things that come back as equivalent.

Reply:
Okay! I will only include the ones that are currently equivalent